### PR TITLE
feat: Windows Defender and Windows version now correctly handled

### PR DESCRIPTION
**Pull request by DennisL68**

### CODE REVIEW

**Comment by PwrKjll** [comment]
Rad 255 kollar ej Win10 LTSC 1809

Rad 341 network path not found

Rad 594 kollar ej om Defender finns

**Comment by DennisL68** [comment]
> Rad 255 kollar ej Win10 LTSC 1809
> 
> Rad 341 network path not found
> 
> Rad 594 kollar ej om Defender finns

594 fixad. 255 förmodligen ok.

File: Get-MiniCompliance.Tests.ps1 [hunk 1]
```diff
@@ -591,56 +596,116 @@ Describe '- Check Security Compliance' -Tag Security {
...
+                if ($WindowsDefenderIsRunning) {
+                    Check -If $MpStatus.AntispywareSignatureAge `
+                        -IsLessOrEqualTo $Compliance.WindowsDefender.Settings.AntiSpywareSignatureMaxAge
+                }
+                if (!$WindowsDefenderIsRunning) {
+                    Set-ItResult -Skipped -Because 'Windows Defender is not running'
+                }
             }
 
             It 'Should check AnitVirus status' {
```
Comment (PwrKjll) [inline Get-MiniCompliance.Tests.ps1]: Antivirus

File: Get-MiniCompliance.Tests.ps1
Comment (PwrKjll) [inline Get-MiniCompliance.Tests.ps1:1]: Rad 346 Using blank passwords, network path not found

File: Get-MiniCompliance.Tests.ps1 (hunk 1 above)
Comment (DennisL68) [reply Get-MiniCompliance.Tests.ps1]: Fixed

**Pull request merged**